pages/common/snort.md

```diff
@@ -0,0 +1,24 @@
+# snort
+
+> Open-source network intrusion detection system.
+> More information: <https://www.snort.org/#documents>.
+
+- Capture packets with verbose output:
+
+`sudo snort -v -i {{interface}}`
+
+- Capture packets and dump application layer data with verbose output:
+
+`sudo snort -vd -i {{interface}}`
+
+- Capture packets and display link layer packet headers with verbose output:
+
+`sudo snort -ve -i {{interface}}`
+
+- Capture packets and save them in the specified directory:
+
+`sudo snort -i {{interface}} -l {{path/to/directory}}`
+
+- Capture packets according to rules and save offending packets along with alerts:
```

Comment:
I am not sure what alerts means here.
LGTM otherwise.
Thank you @SethFalco

Reply:
Happy to revise this, but not sure what would be best without being too verbose.

With Snort, there are different actions you can take on captured packets. One could `block`, `log`, `drop`, `alert`, etc.

For example, in your rules you might `log` all DNS traffic, but `alert` a subset of queried domains, like sites that host malware.

If an `alert` rule is used, 2 files are created instead of just one, a log file, and an alert file. The log file containing all DNS traffic, and the alert file only containing the queries to malware.

https://docs.snort.org/rules/#snort-rule-structure

Reply:
Ahhaaa I understand, probably for a user of snort this would make sense. :)
[I myself had no idea .. so TiL .. thanks]